api/chaos/experimentation/v1/experiment.proto

```diff
@@ -10,17 +10,17 @@ option go_package = "experimentationv1";
 message Experiment {
   enum Status {
     // Unspecifiedf Status.
```

Comment:
Also, can you fix the docstring while you're here? :)

Reply:
fixed